WordPress/Classes/ViewRelated/Reader/ReaderSavedPostsViewController.swift

```diff
@@ -328,7 +323,6 @@ private extension ReaderSavedPostsViewController {
         messageText.replace("[bookmark-outline]", with: icon)
 
         noResultsViewController.configure(title: NoResultsText.noResultsTitle, attributedSubtitle: messageText)
-        noResultsViewController.view.accessibilityLabel = NoResultsText.accessibilityLabel
```

Comment:
It looks like the `NoResultsText.accessibilityLabel` can be removed.

Reply:
Good find! I missed that. Removed in 344c7b0e1ff1fcc255a2663543c2377539b1183a. Will merge once the build is green. Thank you, @ScoutHarris! 🙏